test/ibmq/test_ibmq_job_attributes.py

```diff
@@ -107,6 +107,37 @@ def test_job_name(self, provider):
         self.assertEqual(len(retrieved_jobs), 1)
         self.assertEqual(job_id, retrieved_jobs[0].job_id())
 
+    @requires_provider
+    def test_job_name_update(self, provider):
+        """Test changing the name associated with a job."""
+        backend = provider.get_backend('ibmq_qasm_simulator')
+        qobj = assemble(transpile(self._qc, backend=backend), backend=backend)
+
+        # Use a unique job name
+        initial_job_name = str(time.time()).replace('.', '')
+        job = backend.run(qobj, job_name=initial_job_name, validate_qobj=True)
+        job_id = job.job_id()
+        # TODO No need to wait for job to run once api is fixed
```

Comment:
I keep seeing this in tests, but is this really true? I thought it only applies to `backend.jobs()` with status as the filter?

Reply:
Yeah, that seems to be the case. So they are not needed in the tests, I'll remove them.